torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -336,10 +336,29 @@ def should_skip_lowering(op: torch.fx.node.Node, qconfig_map: Dict[str, QConfigA
     torch.mul: torch.ops.quantized.mul_relu,
 }
 
+def _save_packed_weight(self, destination, prefix, keep_vars):
```

Comment:
just curious, what's the motivation for this?  I would have expected the packed weight to exist in memory, but the unpacked version to be stored on disk.  Is that not how it works anymore?

Reply:
the motivation for this is when saving things to state_dict, by default only parameters and buffers are saved: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L1711-L1716, but packed_weight is not a parameter or buffer

Reply:
looks like we added a `get_extra_state` and `set_extra_state` to `nn.Module`, maybe we could use that as well..

Reply:
oh, my question is why this is saving the packed weight instead of saving the unpacked version?  I thought in all of our workflows the unpacked version is for saving to disk (for historical reasons, for fbgemm vs qnnpack compatibility), and the in-memory version is the only thing that is packed.

or is this code doing that and I'm just not reading it correctly?

Reply:
yeah, what you described is happening as part of the serialization for the packed_weight object itself `torch._C.ScriptObject`, this object is put into state_dict, but the way we serialize this object is to unpack it and save the Tensors

Reply:
got it, thanks for explaining, makes sense!